src/steps/os/openbsd.rs

```diff
@@ -25,9 +25,12 @@ pub fn upgrade_openbsd(ctx: &ExecutionContext) -> Result<()> {
 
     let is_current = is_openbsd_current(ctx)?;
 
-    if ctx.config().dry_run() {
-        println!("{}", t!("Would upgrade the OpenBSD system"));
-        return Ok(());
+    match ctx.config().run_type() {
+        RunType::Dry | RunType::Damp => {
+            println!("{}", t!("Would upgrade the OpenBSD system"));
+            return Ok(());
+        }
+        RunType::Wet => {}
     }
```

Comment:
Since `sudo.execute` handles the dry running behavior just fine, we can remove this part entirely

```suggestion
```

Reply:
(and also that section in the locale file then)